lib/util/openapi.js

```diff
@@ -0,0 +1,235 @@
+'use strict'
```

Comment:
If these utility functions are for use only within the openapi spec, why not put this file in `lib/spec/openapi/`?

Reply:
It is just my personal preference, no special reason for it. I will follow your guide to combine it in single file.

Reply:
Done

Reply:
I wasn't suggesting a single file. I was suggesting something like `lib/spec/openapi/{index.js,utils.js}`.